src/facade/dragonfly_connection.cc

```diff
@@ -240,11 +244,10 @@ void Connection::DispatchOperations::operator()(const MonitorMessage& msg) {
 
 void Connection::DispatchOperations::operator()(const AclUpdateMessage& msg) {
   if (self->cntx()) {
-    for (size_t id = 0; id < msg.username.size(); ++id) {
-      if (msg.username[id] == self->cntx()->authed_username) {
-        self->cntx()->acl_categories = msg.categories[id];
-        self->cntx()->acl_commands = msg.commands[id];
-      }
+    if (msg.username == self->cntx()->authed_username) {
+      self->cntx()->acl_categories = msg.categories;
+      self->cntx()->acl_commands = msg.commands;
+      self->cntx()->keys = msg.keys;
```

Comment:
Actually we can pass `msg` by value and move, can't we? It's owned by us after all (nit because the runtime cost is negligible again)

Reply:
It's either accept msg by value and move the msg.keys or accept my const ref and copy the msg.keys value (as is now). There is not really a difference so I chose the later because it's how the rest of overloads accept their parameters (and looks consistent)

Reply:
> or accept my const ref and copy 

I just meant there is no reason to because it's gonna be dropped after this call either way